nipype/utils/filemanip.py

```diff
@@ -573,13 +573,17 @@ def list_to_filename(filelist):
         return filelist[0]
 
 
+filename_to_list = ensure_list
+list_to_filename = simplify_list
```

Comment:
perhaps we generate warnings for these as well as multiobject to have people start using the new objects.

```python
def filename_to_list(filelist):
    logger.warn('Please use `ensure_list` instead')
    return ensure_list(filelist)
```

Reply:
From https://github.com/nipy/nipype/pull/2517#issuecomment-378368408:

> let's keep warnings out, but update our docs where we reference these objects to the new ones.

I think before we start adding warnings for things we don't plan to drop support for until 2.0, we should try to come up with a coherent plan for warnings and deprecations. While I want to make new names available for people trying to future proof, I don't want to annoy users unduly.

I can open an issue on the subject, where we can try to map out a plan. Seem reasonable?

Reply:
roger.